internal/worker_versioning_rules_test.go

```diff
@@ -139,3 +139,23 @@ func Test_VersioningIntent(t *testing.T) {
 		})
 	}
 }
+
+func Test_WorkerVersioningRules_typed_nil(t *testing.T) {
+	ramp := &VersioningRampByPercentage{
```

Comment:
```suggestion
        var ramp *VersioningRampByPercentage
```
I think this should be equivalent and show what your trying to test a bit clearer

Reply:
Yes, done, I was doing cut/paste from another example...